setup.py

```diff
@@ -1,23 +1,2 @@
-"""
-Setup file for tubes.
-"""
-
-from setuptools import setup, find_packages
-
-setup(
-    name='Tubes',
-    version='0.2.0',
-    description="""
-    Flow control and backpressure for event-driven applications.
-    """,
-    license="MIT",
-    url="https://github.com/twisted/tubes/",
-
-    packages=find_packages(exclude=[]),
-    package_dir={'tubes': 'tubes'},
-    install_requires=[
-        "six",
-        "Twisted",
-    ],
-    include_package_data=True,
-)
+from setuptools import setup
```

Comment:
Note that starting with pip 21.1 you don't even need this boilerplate: https://github.com/pypa/setuptools/pull/2871.

Reply:
Plus it's best to explicitly declare the build backend in `pyproject.toml`.

Reply:
We can have a separate issue for that, I think.